src/Containers/Impl/Docker/LocalDocker.cs

```diff
@@ -2,14 +2,12 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 namespace Microsoft.R.Containers.Docker {
-    public struct LocalDocker {
-        public string Version { get; }
+    public class LocalDocker {
         public string BinPath { get; }
         public string DockerCommandPath { get;}
 
-        public LocalDocker(string binPath, string version, string dockerCommandPath) {
+        public LocalDocker(string binPath, string dockerCommandPath) {
```

Comment:
We did not use Version any where. One problem is that it was not available (easily) if the install location was discovered using service registry entry or if it was constructed from program files environment variable.